python練習用ソースの追加とREADME.mdファイルの内容を修正

Tanpatsu opened the pull request:
### 変更点
- README.mdの内容を修正
- Python_Practice.pyを追加

Tanpatsu committed:
Python練習ソース追加

Python_Practice.pyを新規作成
README.mdを更新

Tanpatsu committed:
python練習ソース作成

Python_Practice.pyの作成
README.mdの更新
addfileの削除

esperantoimo commented on Python_Practice.py:
```diff
@@ -0,0 +1,173 @@
+
+
+# 4.文字列の連結
+comment = 'テスト'
```
文字リテラルの記述は`"`か`'`に統一

esperantoimo commented on Python_Practice.py:
```diff
@@ -0,0 +1,173 @@
+
+# moneyとtotal_priceの比較結果によって条件を分岐
+if money > total_price:
+    print('りんごを' + str(count) +'個買いました')
```
`+`の後ろにスペース
https://github.com/Tanpatsu/test/pull/4/files#diff-1d54066091b5394b45e7468325aca4ecR167
https://github.com/Tanpatsu/test/pull/4/files#diff-1d54066091b5394b45e7468325aca4ecR169

esperantoimo reviewed (changes requested):
以下細かいですが、修正してください
- `Python_Practice.py` のファイル名は何ケースを想定？

Tanpatsu committed:
Update 指摘内容の修正

README.md 最終行に改行を追加
practice.py 命名規約に沿ってファイル名を変更
practice.py 文字リテラルを'に統一
practice.py +の後ろに半角スペース追加

esperantoimo reviewed (approved):
LGTM
